JM/CF/Scripts/3_Game/CommunityFramework/XML/CF_XML.c

```diff
@@ -4,7 +4,7 @@ class CF_XML
 	{
 	}
 
-	static ref CF_XML_Reader Open(string path)
```

Comment:
Are you 100% sure that **ALL** the refs should be removed? (In this file and the other XML related ones)
If so good, otherwise please check if they were needed somewhere.